symengine/llvm_double.cpp

```diff
@@ -143,15 +148,19 @@ std::vector<llvm::Pass *> LLVMDoubleVisitor::create_default_passes(int optlevel)
 #if (LLVM_VERSION_MAJOR < 5)
     passes.push_back(llvm::createLoadCombinePass());
 #endif
+#if (LLVM_VERSION_MAJOR < 7)
     passes.push_back(llvm::createInstructionSimplifierPass());
+#endif
```

Comment:
Can you change this to the workaround in https://patchwork.freedesktop.org/patch/234777/ ?